File: Game Commons/src/de/gymdon/inf1315/game/Spearman.java
package de.gymdon.inf1315.game;

public class Spearman extends Unit {

    public Spearman(Player owner, int x, int y) {
	this.owner = owner;
	this.x = x;
	this.y = y;
	speed = 6;
	range = 1;
	attack = 40;
	defense = 20;
	hp = 100;
	cost = 80;
	combined = 0.1;
	super.resetSpeed();
    }
}

File: Game Commons/src/de/gymdon/inf1315/game/Spearman.kt
package de.gymdon.inf1315.game

class Spearman(owner: Player, x: Int, y: Int) : Unit() {

    init {
        this.owner = owner
        this.x = x
        this.y = y
        speed = 6
        range = 1
        attack = 40
        defense = 20
        hp = 100
        cost = 80
        combined = 0.1
        super.reset()
    }
}
